Containerfile

```diff
@@ -2,6 +2,8 @@ ARG BASE_IMAGE=gcr.io/distroless/static:nonroot
 
 FROM ${BASE_IMAGE}
 
+ENV PLATFORM=container
```

Comment:
How about `FORWARDER_PLATFORM`? Something to make it difficult to overwrite.

Reply:
I don't know the `FORWARDER_PLATFORM` does not make much sense to me. 

We have two situations:
* The `PLATFORM` env var get overwritten when running container - for that we could log warning in the readiness check.
* Our container behavior kicking in in another setting due to presence of the env var - here I think we are good since we must match both key and value (container)  

Let me add some more logging instead.

Reply:
I considered adding sth like this

```
	if p := os.Getenv("PLATFORM"); p != "" {
		if  p == "container" {
			c.apiAddr = "unix:" + cfg.APIUnixSocket
		} else {
			fmt.Fprintf(os.Stderr, "unexpected PLATFORM environment variable: %q\n", p)
		}
	}
```

But I guess it's not really useful.

Reply:
Ugly^

Reply:
Maybe I'm exaggerating, why would somebody add PLATFORM variable to forwarder container.